user/hasher_test.go

```diff
@@ -12,6 +12,22 @@ func TestGenerateUniqueHashWithNoStringsParam(t *testing.T) {
 
 }
 
+func TestGenerateUniqueHashWithOneString(t *testing.T) {
+
+	if _, err := generateUniqueHash([]string{"only one"}, 5); err != nil {
+		t.Fatalf("this should have NOT thrown an error as three strings were given")
+	}
+
+}
+
+func TestGenerateUniqueHashWithOneStringCantBeEmptyString(t *testing.T) {
```

Comment:
As there are other mechanisms for randomness in PR 18, do we even need the first string array.